src/dimension.h

```diff
@@ -97,6 +97,10 @@ typedef struct DimensionInfo
 	int64 interval;
 	int32 num_slices;
 	regproc partitioning_func;
+	// todo:: q:: integer_now_func not needed unless we add an argument to create_hypertable
+	// currently it is nevery populated or used
+	// populating it may make code simpler in some places
+	regproc integer_now_func;
```

Comment:
[DISCUSS]
- [x] should we add a flag in `create_hypertable` to set `integer_now_func`?
if not, this field may be redundant but maintaining it may simplify related code down the road.